tests/js/spec/utils/formatters.spec.jsx

```diff
@@ -2,8 +2,39 @@ import {
   formatFloat,
   formatAbbreviatedNumber,
   formatPercentage,
+  getDuration,
 } from 'app/utils/formatters';
 
+describe('getDuration()', function() {
+  it('should format durations', function() {
+    expect(getDuration(0, 2)).toBe('0.00ms');
+    expect(getDuration(0.1)).toBe('100ms');
+    expect(getDuration(0.1, 2)).toBe('100.00ms');
```

Comment:
Do we want milliseconds to get special treatment here and use the abbreviated version?

Reply:
This scenario was for when you ask for 2 decimal places. I didn't want to change the behavior of the formatter and was looking to better capture how it currently works as these functions had no tests before.

I'm undecided about how to handle time > 0ms and <1s for axis labels. Right now we output `300ms` which is quite a bit larger than 0 and 1s.